src/pal/tests/CMakeLists.txt

```diff
@@ -23,7 +27,7 @@ elseif(PAL_CMAKE_PLATFORM_ARCH_ARM64)
   add_definitions(-DBIT64=1)
   add_definitions(-D_WIN64=1)
 else()
-  message(FATAL_ERROR "Only ARM, ARM64 and AMD64 is supported")
+  message(FATAL_ERROR "Only ARM, ARM64, I386, and AMD64 is supported")
```

Comment:
I'm also confused. build.sh uses x86 and x64 but CMakeLists.txt usually use I386 and AMD64, respectively. 

Here I followed the usual convention in CMakeLists.txt.